db/migrate/20160104041234_create_pieces.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePieces < ActiveRecord::Migration
+  def change
+    create_table :pieces do |t|
+      t.integer :player_id
```

Comment:
I think that a piece belongs to a game, and a game belongs to player 1 and player 2